server/lib/config/pbench-server-default.cfg

```diff
@@ -53,8 +53,7 @@ pbench-local-dir = %(pbench-top-dir)s
 pbench-tmp-dir = %(pbench-local-dir)s/tmp
 
 # pbench-server rest api variables
-bind_host = 0.0.0.0
-bind_port = 8001
+bind_socket=/run/pbench-server.sock
```

Comment:
I wonder if we really need to make this a configuration setting for the server rather than simply defining our NGINX connection this way?

Reply:
You mean not defining it in our config file?

Reply:
Right. We have a lot of old cruft we really wouldn't want anyone to change already. I really think we should probably dump most of this stuff. For example, where we store the archive files now doesn't even matter for deployment because `/srv/pbench` is just a `-v` mount point for the container: they can put it anywhere they want outside, and they just need to know to map it to `/srv/pbench` inside. There's no point to it being a configuration setting.

And this is 100% internal between our NGINX and server: there's no conceivable point to letting anyone change it.

Reply:
Given that the Nginx configuration has to agree with the Pbench Server configuration, I don't think it helps to have the socket location set in the configuration file instead of making it hard-coded.

However, having the value expressed as a literal buried in `shell.py` isn't exactly wonderful either.  That is, `shell.py` should probably be using a symbolic reference to a definition placed in some more appropriate place...which pretty quickly starts looking like the Pbench Server config file.  😆

On the other hand, two lines above where we reference the socket, there's a hard-coded string literal file path for the PID file, which looks quite similar to the path that we're using for the socket...so, with that as prior art, I guess we can just go ahead and hard code the socket path there as well.  😛